GoMeet/src/main/webapp/meeting-event.html

```diff
@@ -21,6 +21,9 @@ <h3>Meeting times here</h3>
       <h2>Meeting Locations</h2>
       <p>Click on the map to propose a new location.</p>
       <div id="map"></div></br>
+      <button onclick="displayPopularLocations()">Most Popular Locations</button><br>
+      <p>Popular Locations:</p>
```

Comment:
Should this be displayed regardless of whether the user has clicked the button above? Maybe reformat to be something more like: 
```
<p>Popular locations:</p> 
<button onclick="displayPopularLocations()">Click to see the most popular locations</button><br>
<ul id="popular-locations-container"></ul> 
```

Reply:
"Popular Locations" is displayed regardless of whether the button is clicked. It is below the button. I thought it looked cleaner having the popular locations heading right above the list. I'm uploading a screenshot but it is taking a while haha.